db/seeds.rb

```diff
@@ -20,10 +20,35 @@
 
 classroom = Classroom.find_or_create_by(name: "Smith's Sixth Grade", school_year: school_year_instance)
 
-user = User.find_or_initialize_by(username: "test")
-if user.new_record?
-  user.password = "password"
-  user.password_confirmation = "password"
-  user.classroom = classroom
-  user.save!
-end
+# Clear existing users to ensure idempotency
+User.destroy_all
+
+# Create users with usernames and admin flag
+User.create!(
```

Comment:
Is there any reason we wouldn't want to use `Teacher.create!` to create the teacher here?
I think it would be good if we used it instead of `User.create` to be explicit about what we're creating.  It also gives us the ability to use things like `is_a?(Teacher)`